pool/mock_queue_test.go

```diff
@@ -55,6 +56,14 @@ func (q *QueueTester) Put(ctx context.Context, j amboy.Job) error {
 	}
 }
 
+func (q *QueueTester) PutMany(ctx context.Context, jobs []amboy.Job) error {
+	catcher := grip.NewBasicCatcher()
+	for _, j := range jobs {
+		catcher.Add(q.Put(ctx, j))
```

Comment:
Can we `catcher.Wrap(q.Put(ctx, j), "job '%s'", j.ID())` so we can track which error produced which error line in the resolved error?

Reply:
done